crates/op-hardforks/src/lib.rs

```diff
@@ -12,7 +12,10 @@ use alloc::vec::Vec;
 use alloy_hardforks::{hardfork, EthereumHardfork, EthereumHardforks, ForkCondition};
 
 pub mod optimism;
-pub use optimism::*;
+pub use optimism::{mainnet::*, sepolia::*};
```

Comment:
```suggestion
pub use optimism::*;
```
this change is unrelated, pls revert to original

Reply:
--> crates/op-hardforks/src/lib.rs:15:9
   |
15 | pub use optimism::*;
   |         ^^^^^^^^^^^ the name `mainnet` in the type namespace is first re-exported here
...
18 | pub use base::*;
   |         ------- but the name `mainnet` in the type namespace is also re-exported here
   |
The clippy is failing if I do this

Reply:
aaah, thanks for explanation!

Reply:
let's do
```rust
pub use optimism::{mainnet as op_mainnet, sepolia as op_sepolia};
```
and
```rust
pub use base::{mainnet as base_mainnet, sepolia as base_sepolia};
```